src/rust/engine/process_execution/src/remote_tests.rs

```diff
@@ -108,10 +108,10 @@ async fn make_execute_request() {
     command_digest: Some(
       (&Digest::new(
         Fingerprint::from_hex_string(
-          "369820f9643feb39980c51fb6d35d59567256946ff3234a371cba8f4de95339c",
+          "c426b29478ec1ddbd872fbfad63ae9151eb9196edcd1a10aa0aab3aa1b48eef8",
```

Comment:
NB: I'm trusting that our implementation is correct in updating these hashes. I simply used what the test now outputs. I did not manually hash things.

Reply:
That is what I have done as well in developing this code.